GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1178,7 +1178,7 @@ private void LoadUnstagedOutput(IReadOnlyList<GitItemStatus> allChangedFiles)
             {
                 var fc = this.FindFocusedControl();
 
-                if (fc == Ok)
+                if (fc is null || fc == Ok)
```

Comment:
If C# 9 will be prepared.

`if (fc is null or Ok)`

Reply:
C#9 can be used
This specific line kind of depends on #8737 (that applies pattern enhancements to many existing snippets)
@RussKie was negative to the change, voting is needed
I prefer it (submitted the PR...) but would still have preferred another parenthesis:
```C#
if (fc is (null or Ok))
```

Reply:
I like the fact that the variable does not need to be repeated anymore.
`if (fc is (null or Ok))` would be most clear for me.
`if (fc is(null or Ok))` would be a compromise for StyleCop and can be easily replaced after an StyleCop update.

Reply:
> I like the fact that the variable does not need to be repeated anymore.

That is my primary reason as well - normally shorter too.

This is what I asked in #8737 - review that and convince @RussKie. He has the majority vote but would not block a common agreement.

Reply:
> was negative to the change, voting is needed

I'm finding it somewhat harder to read, but I won't oppose the change, if we can enforce parens to state the intent clearly.